coderd/insights.go

```diff
@@ -215,6 +216,15 @@ func (api *API) insightsTemplates(rw http.ResponseWriter, r *http.Request) {
 			return xerrors.Errorf("get template insights: %w", err)
 		}
 
+		appUsage, err = tx.GetTemplateAppInsights(ctx, database.GetTemplateAppInsightsParams{
+			StartTime:   startTime,
+			EndTime:     endTime,
+			TemplateIDs: templateIDs,
```

Comment:
Hint: while you are writing unit tests, please make sure that cover paths with `TemplateIDs` and without them (ALL).

Reply:
I will create an improved test-suite as a follow-up to this PR which will perform tests for multiple templates as well.